modules/special-rules/pages/a-dying-race.adoc

```diff
@@ -0,0 +1,16 @@
+= A Dying Race
+
+Every Eldar casualty is a tragedy, one step closer to the race's extinction.
+
+---
+
+Apply these rules to detachments with the {a-dying-race} ability:
+
+* If the detachment is wiped out, immediately remove D3 points from the player's army morale. [either/or]
+* If the detachment is wiped out, the player's army morale immediately goes down by D3 points. [either/or]
```

Comment:
```suggestion
```
Definitely not. Nasty, weak passive case and third person. Yuck. I feel dirty just reading it. >-D

Reply:
Based on the raise/lower idea:
```suggestion
* If the detachment is wiped out, immediately lower the player's army morale by D3 points.
```

Reply:
I deleted this because we're working on the alternate construction in the preceding line, so we don't need it. My comment was just to highlight that this option would've been especially egregious on the passive case and third person front; not only "is destroyed" which we accept as expedient in many cases, but also "goes" and "the player's", which we can avoid through active/second person/imperative as in my suggestion for the alternate option in the preceding line.